src/steps/index.test.ts

```diff
@@ -12,9 +14,38 @@ test('should collect data', async () => {
 
   // Simulates dependency graph execution.
   // See https://github.com/JupiterOne/sdk/issues/262.
-  await fetchAccountDetails(context);
-  await fetchUsers(context);
-  await fetchGroups(context);
+  // await fetchAccountDetails(context);
+  jest
+    .spyOn(owner, 'fetchOwners')
+    .mockImplementation(
+      async ({
+        jobState,
+      }: IntegrationStepExecutionContext<IntegrationConfig>) => {
+        await jobState.addEntity(
+          createIntegrationEntity({
+            entityData: {
+              source: {
+                id: 'owner-1',
+              },
+              assign: {
+                _key: `hubspot-owner-owner-1`,
+                _type: 'acme_user',
+                _class: 'User',
+                // This is a custom property that is not a part of the data model class
+                // hierarchy. See: https://github.com/JupiterOne/data-model/blob/master/src/schemas/User.json
+                firstName: 'Owner',
+                // lastName: owner.lastName,
+                name: `Owner One`,
+                email: 'ownerone@company.tld',
+                username: 'ownerone@company.tld',
+              },
+            },
+          }),
+        );
+      },
+    );
```

Comment:
I don't think this is necessary, but we also haven't talked about tests so far in detail.
In the end, they should look similar to this one for example:
https://github.com/JupiterOne/graph-rapid7/blob/1dc194669abdf5cce8eada59cad1f67c743f0efc/src/steps/index.test.ts

```
describe('Resource we're testing', () => {
    - Recording setup (needs beforeEach, afterEach)...

   test('should collect data', async () => {
         // Your code minus the mockImplementation
         // the await owner.fetchOwners(context) will convert the resource to Entity
         // and it'll be stored in the context.jobState.collectedEntities
   }
}
```

The rest of your testing file looks good :+1: